### DISCUSSION TITLE
Pinout compatibility question.

### DISCUSSION THREAD

**Discussion by andy-novikov** [anarion80/porcupine#3] [General]
Excited to see this project.
I'm currently on corne/zmk/n!n2 with miryoku layout.
Wondering about the status for zmk, on readme it's still not confirmed.

What is the pinout letter routing like? 
can I simply replace my corne with porcupine in my layout config or the switch is more involved?
is the pinout similar to any of zmk supported boards?

**Comment by anarion80** [comment]
Hello. I haven't yet started ZMK config for this... Kinda waiting to properly solder battery/battery connector, but mostly testing my Miryoku-like keymap in QMK plus the keyboard itself (and in here it slowly becomes obvious that pinky stagger is too big and that miryoku is not perfect for fast typing...)

For the pinout, if you're asking if you can use the same shield as corne for ZMK, then without checking in detail I would say no, it's not a drop-in replacement due to different number of thumb keys and outer pinky keys. So that needs to be checked properly.

**Comment by andy-novikov** [comment]
corne is 6x3+3 with option to chip off the extra pinky column to make it 5x3+3 which makes it the regular 36 key.
would have been lovely if porcupine magically just worked in place of corne.

home row mods get in the way of fast typing, I'm expecting [positional hold tap](https://zmk.dev/docs/behaviors/hold-tap#positional-hold-tap-and-hold-trigger-key-positions) to resolve this for me, haven't tested thou yet. or is there something else bothering you on that layout?
